packages/web/tests/components/DiscordCheck/DiscordButtonCheck.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import fetchMock from 'fetch-mock-jest';
+import { DiscordButton } from '../../../src/components/DiscordCheck';
+import { render, screen, waitFor } from '../../testUtils/testTools';
+
+interface User {
+  discordId?: string;
+}
```

Comment:
We should be using the same type the front end uses; adding our own interface adds no value because if the front end's interface changes, the type safety here _**won't**_ catch it